templates/impala/header_title.html

```diff
@@ -1,7 +1,11 @@
 <h1 class="site-title">
 {% macro heading(app, text, icon) -%}
 <a href="{{ url('home') }}" title="{{ _('Return to the {0} Add-ons homepage')|f(request.APP.pretty) }}">
+  {% if waffle.flag ('restyle') %}
+  <img alt="" src="{{ static('img/icons/firefox.png')|safe }}">
```

Comment:
There's text in the a wrapping this so an empty alt attr is ok here unless the alt would add something missing in the text.